frontend/src/views/Lobby.tsx

```diff
@@ -127,6 +157,9 @@ function LobbyPage() {
           ))
         }
       </div>
+      {currentUser?.nickname === host?.nickname
+        ? <PrimaryButton onClick={handleClick}> Start Game </PrimaryButton>
+        : 'Waiting for the host to start the game...'}
```

Comment:
nit: Put `'Waiting for the host to start the game...'` in some sort of React element (`<Text>` maybe, or something slightly larger?). Also, if that change doesn't add a bit more spacing between the names and this text in the "waiting" version, then I'd suggest adding a tad more spacing as well (see image to know what version I'm talking about).
<img width="1216" alt="Current Lobby Page Non-Host Perspective" src="https://user-images.githubusercontent.com/7987279/95028884-a03a5c00-0658-11eb-8ac1-32c3449097f2.png">